.github/workflows/build.yml

```diff
@@ -24,10 +24,13 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install flake8
+          python -m pip install .[dev]
       - name: Lint with flake8
         run: |
           # Stop the build if there are Python syntax errors or undefined names
           flake8 . --count --select=E9,F63,F7 --show-source --statistics --max-line-length=88
           # Full lint, will not stop the build on failure.
           flake8 . --count --exit-zero --ignore=E203,E501,F811 --show-source --statistics --max-line-length=88
+      - name: Test with pytest
+        run: |
+          pytest tests/
```

Comment:
Think you can remove the `tests/` though it's a minor thing. Also make sure we have a newline at the end.